server/src/main/java/org/elasticsearch/action/admin/cluster/node/stats/NodesStatsMetrics.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+package org.elasticsearch.action.admin.cluster.node.stats;
+
+import org.elasticsearch.TransportVersions;
+import org.elasticsearch.action.admin.indices.stats.CommonStatsFlags;
+import org.elasticsearch.common.io.stream.StreamInput;
+import org.elasticsearch.common.io.stream.StreamOutput;
+import org.elasticsearch.common.io.stream.Writeable;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+/**
+ * This class is a container that encapsulates the necessary information needed to indicate which node stats is requested.
```

Comment:
Shall we simplify this sentence a bit?

Something along the lines of, and this is just a suggestion:

> This class encapsulates the metrics and other information needed to define scope when we are requesting node stats. 

If the above makes sense.

Reply:
Thanks. This suggeted sentence is more readable. It is finished in 7f0629e99f46d603d3b3d6ef18bc9fd5a9e0f96e